examples/chef/devices/bridge-app.zap

```diff
@@ -0,0 +1,9473 @@
+{
```

Comment:
Have you tested these device configurations at master?

Reply:
I've only tested for lighting-app, I'll keep only this one in the change.